<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWord</class>
 <widget class="QDialog" name="AddWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a new word</string>
  </property>
  <widget class="QLabel" name="counterLabel">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Counter</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>20</y>
     <width>861</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="foreignlangLineEdit">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="rusLine">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="modeComboBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
